diff --git a/node_modules/list.js/dist/list.js b/node_modules/list.js/dist/list.js
index 640dece..4d4bae5 100644
--- a/node_modules/list.js/dist/list.js
+++ b/node_modules/list.js/dist/list.js
@@ -510,7 +510,7 @@ module.exports = function (list) {
     };
 
     this.matching = function () {
-      return list.filtered && list.searched && item.found && item.filtered || list.filtered && !list.searched && item.filtered || !list.filtered && list.searched && item.found || !list.filtered && !list.searched;
+      return list.filtered && list.searched && item.found && item.filtered || list.filtered && !list.searched && item.filtered || !list.filtered && list.searched && item.found;
     };
 
     this.visible = function () {
